### ISSUE TITLE
Exception 'Bitmap must be premultiplied' 

### ISSUE THREAD

**Issue by michaelvogt** [google-ar/sceneform-android-sdk#276]
When loading a jpg file with `BitmapFactory.decodeFile(textureFilePath);` I get an `java.lang.IllegalArgumentException: Invalid Bitmap: Bitmap must be premultiplied.` thrown by 

`} else if (!bitmap.isPremultiplied()) {
        throw new IllegalArgumentException("Invalid Bitmap: Bitmap must be premultiplied.");`

As far as I understand, premultiplication is only needed when there is an alpha channel, which jpg files usually don't have, right? 

From the Bitmap.isPremultiplied() docs:
> This method only returns true if #hasAlpha() returns true. A bitmap with no alpha channel can be used both as a pre-multiplied and as a non pre-multiplied bitmap.

**Comment by romainguy** [comment]
That is correct, this is a bug in Sceneform. It should check for `hasAlpha() && isPremultiplied()`
